eosdis-nasa/earthdata-pub-api
# Edpub 1255 create note scope table

# Description

Adding new table for limiting note visibility by user/role

## Spec

See Ticket: https://bugs.earthdata.nasa.gov/browse/EDPUB-1255

---

## Validation

1. Make sure all merge request checks have passed (CI/CD).
2. Pull related branches locally.
3. Deploy the api
4. Using your db visualizer of choice, verify that the note_scope table now exists


---

## Change Log

* Created note_scope table
